packages/api-client/src/views/Request/RequestSection/RequestBody.vue

```diff
@@ -73,7 +73,12 @@ const activeExampleContentType = computed(() => {
     return activeExample.value.body.raw.encoding
   }
 
-  return 'none'
+  // Set content type from request if present
+  const contentType = Object.keys(
+    activeRequest.value?.requestBody?.content || {},
```

Comment:
should we start storing content type on the example? Where did we land on example per content type again?

Reply:
lets chat rq

Reply:
its more of a quick fix to be refactored once the above gets in - #4485

Reply:
Ok send it